required/latex-lab/latex-lab-math.dtx

```diff
@@ -316,6 +323,20 @@
 %  |viewer/pane/mathsource=true| (anywhere in the document) and |viewer/pane/mathml=false| (in the
 %  preamble, before the external file is read).  
 %  
+%  
+%  \subsection{Automatic mathml creation with luamml}\label{sec:luamml}
+%  
+%  If lualatex is used, if the package \texttt{unicode-math} is used
+%  and if the package \texttt{luamml} is found\footnote{Until it is 
+%  on CTAN and it must be installed from \url{https://github.com/zauguin/luamml}.}
+%  then it will automatically generate the file |\jobname-luamml-mathml.html| with mathml representations
+%  of all math formulas.
+%  The generation of the file can be suppressed with |math/mathml/luamml=false|.
+%  The package is still quite experimental and the output should be checked, e.g. in
+%  a browser. The file is then used in subsequent compilations and works also with 
```

Comment:
hm, any suggestion how to formulate that?

Reply:
Not really any good idea which is why I didn't suggest a text originally...


perhaps change

> The package is still quite experimental and the output should be checked, e.g. in a browser.

to

> The package is still quite experimental and the output should be checked. The `-luamml-mathml.html` file may be previewed in a browser although you may need to add additional css or javascript declarations to enable browser support for all mathml constructs.



we could additionally adjust the default html file to (say) reference the stix2 opentype math font (improves mathml rendering in chrome a lot, firefox can cope better with legacy fonts)  or potentially reference mathjax to polyfill full mahml3 support, but I'd rather not do either I think. Hopefully the hint that something is needed is enough here, and we'll have some examples available online somewhere that may show what's needed.

in edge for example the image above changes to

![image](https://github.com/user-attachments/assets/f25437c9-8321-4517-b050-cbf41c7cc345)


if you add

```
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_SVG"></script>
```

Reply:
I doubt that manual checking will be sufficient here: but what would I know?  
Maybe that is what @davidcarlisle is saying, inter alia? 

I have some other suggestions on a methodology for improving, and hopefully extending, these abilities.

Reply:
But, without a standard to check against, what can be checked beyond "looks good", which is not the same as "the math information is identical"?

Reply:
@car222222  manual checking obviously dousn't scale if say converting the back catalogue of arxiv, but is a necessary first step and anyway needed in authoring stages of new documents.

"manual" however doesn't mean "by eye". The html format used here was developed while working on the [wtpdf examples](https://github.com/latex3/tagging-project/discussions/72) and allows for example the math to be checked with a suitable screen reader (I used NVDA/MathCAT) at an early stage of the process, and without requiring a pdf reader that can extract the associated files and pass to AT.

Less important with luamml conversions but if using external conversions via tex4ht or latexml etc the other thing that frequently needs correcting and is much easier to do by eye than by an automated test is to check that the correct mathml has been paired up with the correct tex fragment as it's easy to get these out of sync.

Reply:
But do we not really need to check the mml code itself, in detail?  

Rather than just eyeballing the output: does it really matter so much what it looks like? The details of what it actually is are surely much more important, when actually using it, than its prettiness?